core/src/main/java/com/google/errorprone/bugpatterns/AbstractSuppressWarningsMatcher.java

```diff
@@ -73,7 +73,7 @@ protected final Fix getSuggestedFix(AnnotationTree annotationTree) {
       StringBuilder sb = new StringBuilder("@SuppressWarnings({\"" + values.get(0) + "\"");
       for (int i = 1; i < values.size(); i++) {
         sb.append(", ");
-        sb.append("\"" + values.get(i) + "\"");
+        sb.append('"').append(values.get(i)).append('"');
```

Comment:
I'd consider your suggestion to be more readable, as everything is in one line and easier to parse (in my mind, at least). Thanks!